### PR TITLE
fix: usage of `a deprecated Node.js version`

### LINKED ISSUE #13613: `deploy.yml` uses `a deprecated Node.js version`
### Description

https://github.com/rust-lang/rust-clippy/actions/runs/11542544101

### Version

_No response_

### Additional Labels

_No response_

### PR DESCRIPTION
**Pull request by hamirmahal**
Fixes #13613

changelog: none

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -9,58 +9,58 @@ on:
...
 
-    # Generate lockfile for caching to avoid build problems with cached deps
-    - name: cargo generate-lockfile
-      run: cargo generate-lockfile
+      # Generate lockfile for caching to avoid build problems with cached deps
+      - name: cargo generate-lockfile
+        run: cargo generate-lockfile
 
-    - name: Cache
-      uses: Swatinem/rust-cache@v2.7.0
```
Comment (hamirmahal) [inline .github/workflows/deploy.yml:55]: The latest `v2` doesn't use `a deprecated Node.js version`.

**Review by flip1995 — APPROVED** [review]
Thanks, LGTM! Can you revert all the formatting changes, please?

**Comment by hamirmahal** [comment]
You're welcome! I'll take a look at it right now.

**Comment by flip1995** [comment]
Thanks!

@bors r+

**Comment by hamirmahal** [comment]
You're welcome!

**Pull request merged**